optaweb-employee-rostering-backend/pom.xml

```diff
@@ -99,6 +99,10 @@
       <groupId>org.optaplanner</groupId>
       <artifactId>optaplanner-persistence-jackson</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-mvel</artifactId>
+    </dependency>
```

Comment:
Should be a runtime dependency, shouldn't it?

Reply:
Can confirm using scope `runtime` works (the Drools error message didn't mention anything about what scope to add the dependency)